torch/_refs/__init__.py

```diff
@@ -2426,14 +2490,186 @@ def _normalize(
         out (Tensor): normalized tensor.
         mean (Tensor): mean of the tensor along norm_dims.
         rstd (Tensor): 1/std of the tensor along norm_dims.
+        unbiased_var (Tensor): variance of the tensor along norm_dims.
     """
     computation_dtype = utils.get_computation_dtype(a.dtype)
     a_acc = _maybe_convert_to_dtype(a, computation_dtype)
     assert isinstance(a_acc, TensorLike)  # to avoid mypy error for var_mean
-    biased_var, mean = var_mean(a_acc, dim=norm_dims, unbiased=False, keepdim=True)
-    rstd = torch.rsqrt(biased_var + eps)
+    unbiased_var, mean = var_mean(a_acc, dim=norm_dims, unbiased=False, keepdim=True)
+    rstd = torch.rsqrt(unbiased_var + eps)
     out = (a - mean) * rstd
-    return out, mean, rstd
+    return out, mean, rstd, unbiased_var
+
+
+# squeeze backwards from ndims-1 to 0
+def _squeeze_multiple(a: Tensor, dims: List[int]) -> Tensor:
```

Comment:
`prims.squeeze` squeezes multiple times, this helper function is not needed.

Reply:
I thought we were supposed to use the `torch` functions instead of directly calling `prims` / `refs`.

Reply:
I can't dig up the message right now, but using torch functions is not strict. We can use prims/refs if it offers new needed functionality like in this case squeezing multiple times.